cura/Settings/GlobalStack.py

```diff
@@ -344,13 +343,36 @@ def getDefaultFirmwareName(self) -> str:
     def getName(self) -> str:
         return self._metadata.get("group_name", self._metadata.get("name", ""))
 
-    def setName(self, name: "str") -> None:
+    def setName(self, name: str) -> None:
         super().setName(name)
 
     nameChanged = pyqtSignal()
     name = pyqtProperty(str, fget=getName, fset=setName, notify=nameChanged)
 
 
+def getMachinesWithDefinition(definition_id: str, online_only = False) -> List[ContainerStack]:
```

Comment:
Doesn't it make more sense to put this in the machine manager?

Reply:
f000b75661765b3087a7e1fb53c4bd37c3d081e8